lib/taskana-core-test/src/test/java/acceptance/task/create/CreateTaskAccTest.java

```diff
@@ -0,0 +1,967 @@
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_SetPriorityAndDurationCorrectly_When_UsingClassificationOfAttachment()
+      throws Exception {
+    Task task = createDefaultTask();
+    ClassificationSummary classificationSummary =
+        defaultTestClassification()
+            .priority(99)
+            .buildAndStoreAsSummary(classificationService, "admin");
+    ClassificationSummary classificationSummary2 =
+        defaultTestClassification()
+            .priority(99)
+            .buildAndStoreAsSummary(classificationService, "admin");
+    Attachment attachment =
+        TaskAttachmentBuilder.newAttachment()
+            .objectReference(defaultObjectReference)
+            .classificationSummary(classificationSummary)
+            .created(Instant.now())
+            .modified(Instant.now())
+            .received(Instant.parse("2018-01-15T00:00:00Z"))
+            .channel("E-MAIL")
+            .build();
+    Attachment attachment2 =
+        TaskAttachmentBuilder.newAttachment()
+            .objectReference(defaultObjectReference)
+            .classificationSummary(classificationSummary2)
+            .created(Instant.now())
+            .modified(Instant.now())
+            .received(Instant.parse("2018-01-15T00:00:00Z"))
+            .channel("E-MAIL")
+            .build();
+    task.addAttachment(attachment);
+    task.addAttachment(attachment2);
+
+    Task createdTask = taskService.createTask(task);
+
+    assertThat(createdTask.getId()).isNotNull();
+    assertThat(createdTask.getCreator())
+        .isEqualTo(taskanaEngine.getCurrentUserContext().getUserid());
+    Task readTask = taskService.getTask(createdTask.getId());
+    assertThat(readTask).isNotNull();
+    assertThat(createdTask.getCreator())
+        .isEqualTo(taskanaEngine.getCurrentUserContext().getUserid());
+    assertThat(readTask.getAttachments()).isNotNull().hasSize(2);
+    assertThat(readTask.getAttachments().get(1).getCreated()).isNotNull();
+    assertThat(readTask.getAttachments().get(1).getModified()).isNotNull();
+    assertThat(readTask.getAttachments().get(1).getModified())
+        .isEqualTo(readTask.getAttachments().get(0).getCreated());
+    // assertThat(readTask.getAttachments().get(0).getClassification()).isNotNull();
+    assertThat(readTask.getAttachments().get(0).getObjectReference()).isNotNull();
+    assertThat(readTask.getPriority()).isEqualTo(99);
+    assertThat(readTask.getDue()).isNotNull();
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test //
+  void should_ThrowException_When_AttachmentIsInvalid() throws Exception {
+    Consumer<Attachment> testCreateTask =
+        invalidAttachment -> {
+          Task taskWithInvalidAttachment = createDefaultTask();
+          taskWithInvalidAttachment.addAttachment(invalidAttachment);
+          ThrowingCallable call = () -> taskService.createTask(taskWithInvalidAttachment);
+          assertThatThrownBy(call)
+              .describedAs(
```

Comment:
Could you remove describedAs? The reason why the exception is thrown should be clear from the name of the test

Reply:
My bad, this test contains many test cases. Could you separate them into indivildual tests or write a @TestFactory?